R/grayscale.R

```diff
@@ -7,7 +7,33 @@
 #' @export
 #'
 #' @examples
-#' grayscale("images/samples.jpg", "images/greyscale.png")
```

Comment:
please include the library you are using, alone with `library(testit)`

Reply:
Sure. I will do that and also add the unit test cases and raise a PR